src/components/QueryLoader/index.tsx

```diff
@@ -15,10 +15,10 @@ interface Loader {
  * fallback can be used when something goes wrong with the fetch
  */
 const QueryLoader = (props: Loader) => {
-  if (props.fallback) return props.fallback;
-  const { loaderComponent, queryKey, children } = props;
+  const { loaderComponent, queryKey, children, fallback } = props;
   const loader = loaderComponent || <CircularProgress />;
   const isFetching = useIsFetching({ queryKey: [queryKey] });
+  if (fallback && !isFetching) return fallback;
```

Comment:
We could define a default value for when the fallback is not provided. What do you guys think?

Reply:
I think not because we should pass fallback to this component only when an error occurs. This components internally manages only the fetching phase.

Reply:
If the query fails and the fallback is not defined, it will show the Empty state if an empty array is returned.  @mmoio 
I would defined the isError prop separately and then the fallback.

Reply:
I think that the fallback prop is redundant, the same result can be achieved with the following.

```javascript
<QueryLoader>
    {isError? componente1 : componente2}
</QueryLoader>
```